<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pay_loan</class>
 <widget class="QWidget" name="pay_loan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>365</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>365</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>365</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pay loan</string>
  </property>
  <widget class="QLabel" name="receipt_key_text">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>84</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Receipt key</string>
   </property>
  </widget>
  <widget class="QLabel" name="payer_name_text">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>24</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Payer's name</string>
   </property>
  </widget>
  <widget class="QLabel" name="payer_cpf_text">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>54</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Payer's CPF</string>
   </property>
  </widget>
  <widget class="QLabel" name="total_pay_text">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>240</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total to pay</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_message">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>278</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="on_time_output">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>157</y>
     <width>70</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pay_button">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>310</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pay</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>241</x>
     <y>310</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="fees_text">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>160</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>fees</string>
   </property>
  </widget>
  <widget class="QLabel" name="current_installment_text">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>126</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Current Installment</string>
   </property>
  </widget>
  <widget class="QLabel" name="installment_text">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>126</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Installments</string>
   </property>
  </widget>
  <widget class="QPushButton" name="check_button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>55</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: orange;
 border-style: solid;
color: black;
border-radius: 30px</string>
   </property>
   <property name="text">
    <string>Check</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="payer_name_input">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>24</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="payer_cpf_input">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>53</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;CPF only numbers&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="maxLength">
    <number>11</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="receipt_key_input">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>82</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="fees_output">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>157</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="current_installment_output">
   <property name="geometry">
    <rect>
     <x>313</x>
     <y>125</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="total_pay_output">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>240</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="on_time_text">
   <property name="geometry">
    <rect>
     <x>242</x>
     <y>157</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>On time?</string>
   </property>
  </widget>
  <widget class="QLabel" name="installments_output">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>125</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="fees_increase_output">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>158</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="fees_increase_text">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>159</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>fees (+)</string>
   </property>
  </widget>
  <widget class="QLabel" name="total_fees_output">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>191</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="total_fees_text">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>193</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total fees</string>
   </property>
  </widget>
  <widget class="QLabel" name="original_value_output">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>193</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="original_value_text">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>194</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Original value</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>payer_name_input</tabstop>
  <tabstop>payer_cpf_input</tabstop>
  <tabstop>receipt_key_input</tabstop>
  <tabstop>check_button</tabstop>
  <tabstop>pay_button</tabstop>
  <tabstop>cancel_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
